src/Rx/Weno/weno.js

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
this script should be moved to library/js directory

Reply:
done